sklearn/utils/_plotting.py

```diff
@@ -56,3 +58,33 @@ def _validate_from_predictions_params(
         name = name if name is not None else "Classifier"
 
         return pos_label, name
+
+
+def _validate_score_name(score_name, scoring, negate_score):
+    """Validate the `score_name` parameter.
+
+    If `score_name` is provided, we just return it as-is.
+    Otherwise, we infer it from `scoring` by removing `"neg_"` if present and
+    and make it English-readable. If `scoring` is `None`, then we return
+    the default value `"Negative score"` if `negate_score` is `True` and
+    `"Score"` otherwise.
+    """
+    if score_name is not None:
+        return score_name
+    elif scoring is None:
+        return "Negative score" if negate_score else "Score"
+    else:
+        score_name = scoring.__name__ if callable(scoring) else scoring
+        if negate_score and score_name.startswith("neg_"):
+            score_name = score_name[4:]
+        return score_name.replace("_", " ").capitalize()
+
+
+def _compute_scale_type_ratio(data):
```

Comment:
Let's use a more descriptive name:

```suggestion
def _interval_max_min_ratio(data):
```

Reply:
If this suggestion is accepted then `test_compute_scale_type_ratio` should be updated accordingly.